processing/src/main/java/org/apache/druid/query/groupby/epinephelinae/vector/GroupByVectorColumnSelector.java

```diff
@@ -27,6 +27,7 @@
 {
   int getGroupingKeySize();
 
+  @SuppressWarnings("unused")  // false positive unused inspection warning for "keySize"
```

Comment:
Could you please create an issue in youtrack.jetbrains.com and link here?

Reply:
Added https://youtrack.jetbrains.com/issue/IDEA-231034